tests/parametric/conftest.py

```diff
@@ -886,6 +962,10 @@ def test_server(
         "DD_TRACE_AGENT_PORT": test_agent_port,
         "APM_TEST_CLIENT_SERVER_PORT": apm_test_server.port,
     }
+    test_server_env = {}
```

Comment:
where is this dict used ?

Reply:
🤔 oh I was trying to not set env variables that are `None` but it looks like I didn't actually use it on like 998. Good catch, I will fix it to do that!